doc/articles/toc.yml

```diff
@@ -51,7 +51,8 @@
           items:
             - name: Overview
               href: tutorials-intro.md
-
+            - name: Create a blank Uno App using the Wizard
+              href: getting-started/wizard/using-wizard.md  
```

Comment:
We should be referencing this new page from the getting started, maybe as a info bubble?

Reply:
https://github.com/unoplatform/uno/pull/12095/commits/fc8c18f6dbb387802d609af1633f60e81ad98f70#diff-9bc3ab0b6409b21f18f3193e38a11cd28329ee3b72800f95ca952041838a2af1R74-R76